pkgs/development/python-modules/libuuu/default.nix

```diff
@@ -0,0 +1,64 @@
+{
+  lib,
+  stdenv,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  setuptools-scm,
+  autoPatchelfHook,
+  pytestCheckHook,
+  udev,
+}:
+
+buildPythonPackage rec {
+  pname = "libuuu";
+  version = "1.5.182";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-k6JwGxYeFbGNl7zcuKN6SbRq8Z4yD1dXXL3ORyGqhYE=";
+  };
+
+  build-system = [
+    setuptools
+    setuptools-scm
+  ];
+
+  nativeBuildInputs = lib.optionals stdenv.isLinux [
+    autoPatchelfHook
+  ];
+
+  buildInputs = lib.optionals stdenv.isLinux [
+    udev
+  ];
+
+  dependencies = [
+    setuptools-scm
+  ];
+
+  pythonImportsCheck = [
+    "libuuu"
+  ];
+
+  # Prevent tests to load the plugin from the source files instead of the installed ones
+  preCheck = ''
+    rm -rf libuuu
+  '';
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
+
+  meta = {
+    description = "A python wraper for libuuu";
+    homepage = "https://pypi.org/project/libuuu/";
+    license = lib.licenses.bsd3;
+    maintainers = with lib.maintainers; [ GaetanLepage ];
+    badPlatforms = [
+      # The pypi archive does not contain the pre-built library for these platforms
```

Comment:
Why not build it from source then?

Reply:
Lack of time :/ I pinged the maintainer of `pkgs.uuu` in an earlier comment...

Reply:
@GaetanLepage, could you create the expression for the packaging? I can certainly review it, but I do not have the time to look at making that myself.

Reply:
Me neither... It doesn't look that straight-forward.
Do you think it could be done from `pkgs.uuu`'s derivation or we should do a separate `pkgs.libuuu` one ?

Reply:
I believe you could create a new package and use the existing one to provide the source. So, we would update them together. I don't know if anyone has other thoughts about that?

Reply:
I checked and I think that it should belong in the main `uuu` package.
The issue is that the shared libraries are not built currently.
I tried uncommenting [this line](https://github.com/nxp-imx/mfgtools/blob/6c2141e3e820c5a487482adcde5cf66266c07656/libuuu/CMakeLists.txt#L70) in `libuuu`'s `CMakeLists.txt` but got a build error.
I don't have much more time to spend on this unfortunately.

@SuperSandro2000 would you be fine if we go with the pre-built approach .